### PR TITLE
New Feature: Trip plan

### PR DESCRIPTION
**Pull request by TaninDean**
[Build Status](https://app.travis-ci.com/ThaiRepose/thairepose) [codecov](https://codecov.io/gh/ThaiRepose/thairepose)
### What is the feature?
Trip plan for user to create blot with feature:
 * Create blog with detail of trip
 * Category
 * Edit blog
 * View blog
 * Delete blog
 * Like blog
### What is the solution?
Create models of trip plan to store user blog
### How to test?
Unittest for simulate fuction in TripPlan
### Screen sho
สกรีนช็อต 2021-11-09 114858
สกรีนช็อต 2021-11-09 114824
t
สกรีนช็อต 2021-11-09 114910
สกรีนช็อต 2021-11-09 114922
สกรีนช็อต 2021-11-09 114935

### CODE REVIEW

File: trip/models.py [hunk 1]
```diff
@@ -1,6 +1,21 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.urls import reverse
+from ckeditor.fields import RichTextField
+
+
+class CategoryPlan(models.Model):
+    """Extended user model class that use for Category of Trip plan.
```
Comment (nabhan-au) [inline trip/models.py:8]: I think this function doesn't extend from user model

File: trip/models.py [hunk 2]
```diff
@@ -23,6 +50,10 @@ def get_absolute_url(self):
         """Return redirect to all trip pages."""
         return reverse("trip:tripdetail", args=(str(self.id)))
 
+    def total_like(self):
+        """Return number of count."""
+        return self.like.count()
+
 
 class Review(models.Model):
     """Extended user model class that use for Review.
```
Comment (nabhan-au) [inline trip/models.py:59]: I think this function doesn't extend from user model

File: trip/models.py (hunk 1 above)
Comment (TaninDean) [reply trip/models.py]: oh my bad. I was fix it already

**Review by nabhan-au — COMMENTED** [review]
In view.py, Docstring should be in google style

**Comment by nabhan-au** [comment]
Capture
I think post date attribute in TripPlan model it should have default value

**Comment by nabhan-au** [comment]
Capture1
Duration and price shouldn't be negative value

**Comment by TaninDean** [comment]
i update docstring and set min value already but for default in post_date my django go show error the auto_now_add is create a dafault so you dont need to set i
สกรีนช็อต 2021-11-09 150222
t.

**Review by nabhan-au — APPROVED** [review]
Nice work!!

**Pull request merged**